app/assets/stylesheets/pghkids.scss

```diff
@@ -0,0 +1,40 @@
+@import url('http://fonts.googleapis.com/css?family=Open+Sans:300,400,700,800,600');
+// from bower_components
+@import "breakpoint-sass/stylesheets/breakpoint";
+
+@import "library/reset";
+@import "library/grid";
+// @import "library/variables";
```

Comment:
Cool. I think the most important thing is consistency more than specifically what folder it's in. If you don't mind moving the other ones at some point in the future :+1:.